File: problem-solving/algorithms/apple-and-orange/apple-and-orange.js
function countApplesAndOranges(s, t, a, b, apples, oranges) {
    const appleCount = apples.filter(apple => a + apple >= s && a + apple <= t).length;
    const orangeCount = oranges.filter(orange => b + orange >= s && b + orange <= t).length;

    console.log(appleCount);
    console.log(orangeCount);
}

File: problem-solving/algorithms/apple-and-orange/apple_and_orange.py
def countApplesAndOranges(s, t, a, b, apples, oranges):
    apple_count = sum([1 for apple in apples if a + apple >= s and a + apple <= t])
    orange_count = sum([1 for orange in oranges if b + orange >= s and b + orange <= t])

    print(apple_count)
    print(orange_count)